opencodelists/management/commands/setup_local_dev_databases.py

```diff
@@ -0,0 +1,103 @@
+from django.core.management import BaseCommand, call_command
+
+from coding_systems.versioning.models import update_coding_system_database_connections
+from opencodelists import settings
+from opencodelists.tests.fixtures import build_fixtures
+
+
+class Command(BaseCommand):
+    help = """Populates local opencodelists dbs with test fixture data: Loads the CodingSystemReleases needed for the snomed and dmd fixtures into the Core db (you will be prompted for consent to do this), sets up new coding system release dbs, populates them with test fixture data, instantiates a universe of fixture objects, and creates a superuser."""
+
+    def handle(self, *args, **options):
+        load_versioning_fixture_answer = input(
+            "Running this command will load the CodingSystemReleases needed for the snomed and dmd test fixtures, into the Core db. Do you want to proceed? (Y/n)"
+        )
+
+        if load_versioning_fixture_answer == "n":
+            return
+
+        call_command(
+            "loaddata",
+            "coding_systems/versioning/fixtures/coding_system_releases.json",
+        )
+
+        database_paths = find_dbs(settings.CODING_SYSTEMS_DATABASE_DIR)
```

Comment:
Do you need to delete _all_ coding system database file, or just the ones which will conflict with the `migrate` command calls below? Could you work out what the destination paths for those calls are (e.g. from the `database` kwarg) and just delete those instead?